distributed/cli/dask_worker.py

```diff
@@ -216,6 +216,14 @@
 @click.argument(
     "preload_argv", nargs=-1, type=click.UNPROCESSED, callback=validate_preload_argv
 )
+@click.option(
+    "--preload-nanny",
+    type=str,
+    multiple=True,
+    is_eager=True,
+    help="Module that should be loaded by each nanny "
+    'like "foo.bar" or "/path/to/foo.py"',
+)
```

Comment:
Currently `preload_nanny_argv` can't be specified on the command line. Do you have any thoughts on how we could support a `--preload-nanny-argv` CLI option? To be clear, I suspect this may be difficult with our current `click` setup and don't think the lack of a `--preload-nanny-argv` option should stop us from merging this PR

Reply:
Yeah, I wasn't sure how to pass things through from click.  My sense is that we probably have to point users to the configuration to handle this.  

I've added https://github.com/dask/dask/pull/6077 to help with documentation here.